src/components/primitive/Icons/icon.component.props.ts

```diff
@@ -0,0 +1,11 @@
+import { IconType } from './icons';
+
+interface IconProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  icon: IconType;
+  size?: number;
+  color?: string;
```

Comment:
Allow colors that are there in the colors file and not any string.

Reply:
replaced colors- string with ColorType.